integrations/sonarqube/README.md

```diff
@@ -0,0 +1,17 @@
+# SonarQube integration
```

Comment:
This should probably be placed in the `docs/` folder, that way, it will also show up on the website :)

Reply:
> This should probably be placed in the `docs/` folder, that way, it will also show up on the website :)

The `docs/` folder is specific to Stryker, and I was trying to keep this jq filter independent of Stryker as it can be used with reports from any mutation testing framework that uses the schema in this repository. But I've now moved it into the `docs/` folder anyway, as having this on the Stryker website would make it much more discoverable which is probably more important. Are any other changes required to get this published to the website?